### PR TITLE
+oasis.0.4.9

### PR DESCRIPTION
**Pull request by gildor478**

### CODE REVIEW

**Comment by fdopen** [comment]
Why is `oasis.0.4.9` incompatible with the latest version of ocamlbuild, but `oasis.0.4.8` not?
Really a regression? Or wrong dependencies in `oasis.0.4.9` or `oasis.0.4.8` ?

**Comment by gildor478** [comment]
The opam file is generated by oasis2opam and the ocamlbuild line is due to the output of this command:

$ oasis2opam --query ocamlbuild
"ocamlbuild" is provided by: ("ocamlbuild" {= "0.9.3"} | "ocamlbuild" {= "0.9.2"} | "ocamlbuild" {= "0.9.1"} | "ocamlbuild" {= "0.9.0"} | "ocamlbuild" {= "0"})

@Chris00 why the package doesn't simply depend on ocamlbuild?

Fixing it in the meantime.

File: packages/oasis/oasis.0.4.9/url [hunk 1]
```diff
@@ -0,0 +1,2 @@
+archive: "https://forge.ocamlcore.org/frs/download.php/1690/oasis-0.4.9.tar.gz"
```
Comment (avsm) [inline packages/oasis/oasis.0.4.9/url]: It looks like the forge.ocamlcore.org has a TLS certificate issue, preventing installation on some distros:

```
/ # curl -OL https://forge.ocamlcore.org/frs/download.php/1690/oasis-0.4.9.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (60) SSL certificate problem: unable to get local issuer certificate
More details here: https://curl.haxx.se/docs/sslcerts.html

```

Comment (gildor478) [inline packages/oasis/oasis.0.4.9/url]: I have replaced the SSL certificate yesterday. Hopefully should be OK now.

Anyway, I have released 0.4.10 a few minutes ago so this PR is obsolete.

**Pull request closed without merging**